pandas/core/computation/check.py

```diff
@@ -7,7 +7,8 @@
 try:
     import numexpr as ne
     ver = ne.__version__
-    _NUMEXPR_INSTALLED = ver >= LooseVersion(_MIN_NUMEXPR_VERSION)
+    _NUMEXPR_INSTALLED = LooseVersion(ver) >= \
+        LooseVersion(_MIN_NUMEXPR_VERSION)
```

Comment:
Please use (..) for line continuation instead of `\` (but here maybe you can also add the 'LooseVersion' on the line above (`ver = LooseVersion(ne.__version__)` to avoid the need of a line continuation)

Reply:
Was trying to keep all LooseVersion comps on the same line, but changed this back to put the one ``LooseVersion`` back on the line above for readability.